tests/transpiler/qiskit/test_coverage_from_qiskit.py

```diff
@@ -31,6 +31,11 @@ def is_package_installed(package_name: str) -> bool:
 ALL_TARGETS = [("braket", 0.98), ("cirq", 0.98), ("pyquil", 0.98), ("pytket", 0.98)]
 AVAILABLE_TARGETS = [(name, version) for name, version in ALL_TARGETS if is_package_installed(name)]
 
+# TODO: Update Pyqasm with the definitions of these gates
+GATES_TO_SKIP = {
+    "C4XGate",
+}
```

Comment:
Make sure to turn into GitHub issue

Reply:
Sorry, forgot to update this but this gate is also passing now!